Cargo.toml

```diff
@@ -19,6 +19,25 @@ license = "MIT/Apache-2.0"
 travis-ci = { repository = "amethyst/amethyst", branch = "master" }
 
 [features]
+default = ["animation", "audio", "locale", "network"]
+
+animation = [
+    "amethyst_animation"
+]
+audio = [
+    "amethyst_audio"
+]
+gltf = [
+    "amethyst_gltf",
+    "amethyst_animation"
```

Comment:
not totally positive but can we not reference the `animation` feature instead of the crate here?

Reply:
doesn't work